suite-native/accounts/src/useAccountForm.ts

```diff
@@ -0,0 +1,18 @@
+import { yup } from '@trezor/validation';
+import { useForm } from '@suite-native/forms';
+
+export const MAX_ACCOUNT_LABEL_LENGTH = 30;
+
+export type AccountFormValues = yup.InferType<typeof accountFormValidationSchema>;
+
+const accountFormValidationSchema = yup.object({
+    accountLabel: yup.string().required().max(MAX_ACCOUNT_LABEL_LENGTH),
+});
+
+export const useAccountForm = (accountName?: string) =>
```

Comment:
Please unify naming use `label` or `name` everywhere. I think `label` is already more widely used across codebase.

Also better name for this hook should be `useAccountLabelForm`

Reply:
changed in 8fdbadc